TerminatorPlus-API/src/main/java/net/nuggetmc/tplus/api/utils/ChatUtils.java

```diff
@@ -1,20 +1,20 @@
 package net.nuggetmc.tplus.api.utils;
 
-import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.minimessage.MiniMessage;
 
 import java.text.NumberFormat;
 import java.util.Locale;
 
 public class ChatUtils {
-    public static final String LINE = NamedTextColor.GRAY + "------------------------------------------------";
+    public static final String LINE = MiniMessage.miniMessage().deserialize("<gray>------------------------------------------------").toString();
     public static final String BULLET = "\u25AA";
-    public static final String BULLET_FORMATTED = NamedTextColor.GRAY + BULLET + NamedTextColor.WHITE;
-    public static final String EXCEPTION_MESSAGE = NamedTextColor.RED + "An exception has occured. Please try again.";
+    public static final String BULLET_FORMATTED = MiniMessage.miniMessage().deserialize("<gray>" + BULLET + "<white>").toString();
+    public static final String EXCEPTION_MESSAGE = MiniMessage.miniMessage().deserialize("<red>An exception has occured. Please try again.").toString();
```

Comment:
MiniMessage returns `Component`. Calling toString on Components will not work.
Please test this in game!!